react/w-books/src/app/screens/Home/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+
+// import { useRequest } from 'app/hooks/useRequest';
+import BookList from '~components/BookList';
+import NavBar from '~components/NavBar';
+// import { getBooks } from '~services/booksService';
+
+import { BOOKS_URL } from '~constants/api';
+
+import styles from './styles.module.scss';
+
+function Home() {
+  const [books, setBooks] = useState();
+  useEffect(() => {
+    const headers = {
+      'access-token': localStorage.getItem('access-token'),
+      uid: localStorage.getItem('uid'),
+      client: localStorage.getItem('client')
+    };
+    axios.get(BOOKS_URL, { params: headers }).then(r => {
+      setBooks(r.data.page);
+    });
```

Comment:
Don't use axios, by default use useRequestHook, for the header you can setup this in the file API.js config file